tiledb/sm/rest/rest_client_remote.cc

```diff
@@ -1516,4 +1516,23 @@ RestClientRemote::post_consolidation_plan_from_rest(
       serialization_type_, returned_data);
 }
 
+const RestCapabilities& RestClientRemote::get_capabilities_from_rest() {
```

Comment:
This function is not thread-safe. You can either guard it with a mutex ~~or change `rest_capabilities_` to an `atomic<unique_ptr<RestCapabilities>>` and initialize it by~~ [never mind](https://stackoverflow.com/questions/35067675/atomic-operations-on-unique-ptr).